praw/exceptions.py

```diff
@@ -39,7 +39,7 @@ class ClientException(PRAWException):
 class WebSocketException(ClientException):
     """Indicate exceptions caused by use of WebSockets."""
 
-    def __init__(self, message, exception):
+    def __init__(self, message: str, exception: Exception):
```

Comment:
Is there a lint-type test we can run to ensure all methods have type signatures going forward?

Reply:
Not that I know of, but you can try looking into MyPy or something similar

Reply:
And plus, not all files have types yet.